index.html

```diff
@@ -379,6 +385,58 @@
   </dl>
 </section>
 
+<section>
+  ## Events
+
+  ```webidl
+    dictionary SerialConnectionEventInit : EventInit {
+      required SerialPort port;
+    };
+
+    [Exposed=(DedicatedWorker,Window), SecureContext]
+    interface SerialConnectionEvent : Event {
+      constructor(DOMString type, SerialConnectionEventInit eventInitDict);
+      [SameObject] readonly attribute SerialPort port;
+    };
+  ```
+
+  <pre class="example js" title="Keep UI in sync with connected and disconnected serial ports">
+    navigator.serial.addEventListener('connect', event => {
+      // Add |event.port| to the UI.
+    });
+
+    navigator.serial.addEventListener('disconnect', event => {
+      // Remove |event.port| from the UI.
+    });
+  </pre>
+
+  ### Serial port connection
+
+  When the UA detects a new serial port connected to the host it MUST
+  perform the following steps for each script execution environment:
+
+  1. Let |port| be the {{SerialPort}} object representing the new serial port
```

Comment:
As in `requestPort()` we haven't been very specific about what it means to have access to a port but we could at least reference the "Security considerations" section as we do there.

Reply:
Fixed in https://github.com/reillyeon/serial/pull/5/commits/b507ba92a36a753af6cd290c2a7e45e031867377